airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -153,7 +153,17 @@ def _group_logs_by_host(self, logs):
         grouped_logs = defaultdict(list)
         for log in logs:
             key = getattr(log, self.host_field, 'default_host')
-            grouped_logs[key].append(log)
+            
+            try:
+                grouped_logs[key].append(log) 
+            except TypeError as e:
+                if not isinstance(key, Hashable): 
+                    raise ValueError("The host field in all log records needs to be hashable. "
+                    "If you are using filebeat, read here: "
+                    "https://github.com/apache/airflow/issues/15613#issuecomment-1104487752") from e
```

Comment:
Instead of linking to GitHub, we should have documentation for this.

Reply:
Actually better solution will be to copy the explanation to our ElasticSearch documentation (at airflow.apache.org) and link from it to there. The error message should explain the reason and link to the detailed discussion/explanation why - but linking to an issue is only fine only in a source comment, rather than in a user message. Theree we should only link to a documentation we control.

Reply:
Get your point, just got set up with breeze to write some proper documentation. 

I have a question: `airflow.providers.elasticsearch.log.es_task_handler.ElasticsearchTaskHandler` has `offset_field` and `host_field` paramaters in its constructor. I have a hard time figuring out where these are being set / come from. Are they configurable?

Reply:
Actually - I know that - (Re-read this) - I believe each logging handler can be configured with parameters - you can read it in "logging" configugration in our docs./